decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -99,8 +99,8 @@ namespace :decidim do
         if decidim_gemspec.version.to_s =~ /\.dev$/
           # With the .dev version the package does not exist at NPM yet.
           {
-            dev: "https://gitpkg.now.sh/mainio/decidim/packages_dev/dev?feature/split-npm-packages",
-            prod: "https://gitpkg.now.sh/mainio/decidim/packages_dev/all?feature/split-npm-packages"
+            dev: "https://gitpkg.now.sh/decidim/decidim/packages_dev/dev",
+            prod: "https://gitpkg.now.sh/decidim/decidim/packages_dev/all"
```

Comment:
Branch is missing here, add `?develop` to the end of the URLs.

Same thing for the URLs in the `package.json` files.